docs/admin_guide.md

```diff
@@ -908,7 +908,7 @@ With the following content:
 
 ```ini
 [program:messenger]
-command=php /var/www/mbin/bin/console messenger:consume async async_ap failed --time-limit=3600
+command=php /var/www/mbin/bin/console messenger:consume old async outbox deliver inbox receive resolve failed --time-limit=3600
```

Comment:
This means that `old` is the most important queue, then async and outbox. And finally inbox, receive, resolve and failed and queues with less prio.

We could still create multiple different messenger:consume commands, as long as you do not use the same messenger/queue names across multiple `messenger:consume` you should be good. 

For example n supervisor we could create a `messenger:consume` for example: "async + old + inbox + ...". And other messenger:consume for let's say "deliver + outbox + receive.".  Again just an example. This could help us prioritize the work better and do jobs in parallel if needed.

Reply:
why I want to stay at one consume command with all the queues in it:
- prioritization is taken care of. That means that **all** your messengers will prioritize the queues the way we tell them
- simplicity: you don't have multiple supervisor configs to worry about or multiple docker containers
- better worker utilization: if you have 3 consume commands all working on different queues then you have events where 
  - 2/3 of your messengers are idling while the third one is struggling to keep up
  - all of them are running at once and you defined more workers than your system should handle because usually they are not running at the same time and now your server is slow